src/app/clusters/microwave-oven-control-server/microwave-oven-control-server.h

```diff
@@ -85,8 +83,9 @@ class Instance : public CommandHandlerInterface, public AttributeAccessInterface
     Clusters::OperationalState::Instance & mOpStateInstance;
     Clusters::ModeBase::Instance & mMicrowaveOvenModeInstance;
 
-    uint32_t mCookTimeSec        = kDefaultCookTimeSec;
-    uint8_t mSupportedWattLevels = 0;
+    uint32_t mCookTimeSec                = kDefaultCookTimeSec;
+    uint8_t mSupportedWattLevels         = 0;
```

Comment:
Isn't mSupportedWattLevels unused now?  It's certainly not assigned to anymore...